### PR TITLE
Created User Category Nav

### PR DESCRIPTION
**Pull request by strawberrie68**
Created CSS for User category Nav 
- component should be able to toggle between different sections in the profile, later on

profilecategory

### CODE REVIEW

File: src/components/UserCategoryNav.jsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
+import {GridFour, CookingPot, NotePencil, Brain} from "@phosphor-icons/react";
+
+const UserCategoryNav = () => {
+  return (
+    <div className="flex gap-8">
+      <div className="flex  flex-col ">
+        <div className="bg-black w-full h-1 "></div>
+        <div className="flex flex-row gap-2 text-xxs font-semibold items-center mt-2 w-3/4 px-2">
+          <div>
```
Comment (laurawong) [inline src/components/UserCategoryNav.jsx:10]: Is this extra div required?

File: src/components/UserCategoryNav.jsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
+import {GridFour, CookingPot, NotePencil, Brain} from "@phosphor-icons/react";
+
+const UserCategoryNav = () => {
+  return (
+    <div className="flex gap-8">
+      <div className="flex  flex-col ">
```
Comment (laurawong) [inline src/components/UserCategoryNav.jsx]:
```suggestion
      <div className="flex flex-col ">
```
Remove extra spaces in this file

File: src/components/UserCategoryNav.jsx [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+        <div className="bg-black w-full h-1 "></div>
+        <div className="flex flex-row gap-2 text-xxs font-semibold items-center mt-2 w-3/4 px-2">
+          <div>
+            <GridFour size={20} />
```
Comment (laurawong) [inline src/components/UserCategoryNav.jsx]: Use constant for size

**Review by laurawong — APPROVED** [review]

File: src/components/UserCategoryNav.jsx (hunk 1 above)
Comment (strawberrie68) [reply src/components/UserCategoryNav.jsx:10]: Yes. It's not functioning right now. but when the link is selected it should have a black bar above it

**Comment by strawberrie68** [comment]
profilecategory

File: cypress/component/UserCategoryNav.cy.jsx [hunk 4]
```diff
@@ -0,0 +1,11 @@
+import UserCategoryNav from "../../src/components/UserCategoryNav";
+
+describe("UserCategoryNav.cy.jsx", () => {
```
Comment (laurawong) [inline cypress/component/UserCategoryNav.cy.jsx]: Test suite name should be human readable

File: cypress/component/UserCategoryNav.cy.jsx [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+  it("playground", () => {
```
Comment (laurawong) [inline cypress/component/UserCategoryNav.cy.jsx]: What does playground mean?

File: cypress/component/UserCategoryNav.cy.jsx [hunk 6]
```diff
@@ -0,0 +1,11 @@
+import UserCategoryNav from "../../src/components/UserCategoryNav";
+
+describe("Render <UserCategoryNav /> component", () => {
```
Comment (laurawong) [inline cypress/component/UserCategoryNav.cy.jsx]:
```suggestion
describe("Render UserCategoryNav component", () => {
```

File: src/tests/UserCategoryNav.test.js [hunk 7]
```diff
@@ -0,0 +1,19 @@
+import {render, screen} from "@testing-library/react";
+import "@testing-library/jest-dom";
+import UserCategoryNav from "../components/UserCategoryNav";
+
+test("renders UserCategoryNav and checks for elements", () => {
```
Comment (laurawong) [inline src/tests/UserCategoryNav.test.js]: You can delete this file then, since it's already covered.

**Review by laurawong — APPROVED** [review]

File: cypress/component/UserCategoryNav.cy.jsx [hunk 8]
```diff
@@ -0,0 +1,11 @@
+import UserCategoryNav from "../../src/components/UserCategoryNav";
+
+describe("Render UserCategoryNav  component", () => {
```
Comment (laurawong) [inline cypress/component/UserCategoryNav.cy.jsx:3]:
```suggestion
describe("Render UserCategoryNav component", () => {
```

**Review by laurawong — APPROVED** [review]

**Pull request merged**